Add 2019 census results blog post (#111)
reactiflux/reactiflux.com: src/blog/2019-census/index.md

Those with negative feedback, broadly, had had a bad experience with another member. We're glad to get word of these incidents, but our options for responding to these reports—weeks or months after the altercation—are limited. We're taking this as a signal that we need stronger mechanisms to report bad behavior within the server.

We also received a small number of responses that expressed their frustrating with what they felt was a clique of Reactiflux insiders, both within the MVP/Mod/Admin group and regulars with no specific role. This is a concern of ours, but one that we feel we have to balance with being friendly: we want to avoid cliques that exclude others, but still allow people to be friends within the context of Reactiflux. Any group that has been bound by a single intent for multiple years—in this case, answering programming questions—is bound to develop some internal social ties and places of friction, but we want to keep that circle open.

## Where we stand

> We're dismayed to see how heavily male-skewed the population is. While digging into other demographic surveys, though, we found these results are mirrored in other large communities.

Author: Sonic brought up that this reads as though we were shocked by it, but it wasn't really a surprise. That's a good point.

```suggestion
While not surprising, we're not happy with how heavily male-skewed the population is. While digging into other demographic surveys, though, we found these results are mirrored in other large communities.
```
